### ISSUE TITLE
svelte3/ignore-styles not working

### ISSUE THREAD

**Issue by Luke-zhang-04** [sveltejs/eslint-plugin-svelte3#114]
My config looks like this:
```js
    overrides: [
        {
            files: ["*.svelte"],
            parser: "@typescript-eslint/parser",
            parserOptions: {
                tsconfigRootDir: __dirname,
                project: ["./tsconfig.json"],
                extraFileExtensions: [".svelte"],
            },
            plugins: ["@typescript-eslint", "prefer-arrow", "svelte3"],
            processor: "svelte3/svelte3",
            rules: {
                "svelte3/ignore-styles": 0,
            },
        },
    ],
```

But I'm still getting a lint error
```
22:10  error  Identifier is expected  ParseError
```

Specifically, it seems like it doesn't like the nested class
```scss
    .swiper-container {
        width: 100%;
        height: min(100vh, 100vw);
        max-height: max-content;
        position: relative;

        .swiper-wrapper {
            position: absolute;
        }
    }
```

And for `svelte3/ignore-styles`, I've tried setitng it to 0, 1, and 2. If I try to set it to a function, I get an error
```
Error: .eslintrc.js#overrides[0]:
        Configuration for rule "svelte3/ignore-styles" is invalid:
        Severity should be one of the following: 0 = off, 1 = warn, 2 = error (you passed '[Function: log]').
```

**Comment by dummdidumm** [comment]
See https://github.com/sveltejs/eslint-plugin-svelte3#svelte3ignore-styles . There's nothing that can be done except ignoring the styles right now because eslint requires plugins to be synchronous, but preprocessors can be asynchronous.

**Comment by Luke-zhang-04** [comment]
Hm ok thanks it looks like it's supposed to be a setting not a rule. Thanks.
